packages/cli-lib/templates.js

```diff
@@ -39,9 +53,24 @@ const getAnnotationValue = (annotations, key) => {
  */
 const isCodedFile = filePath => TEMPLATE_EXTENSION_REGEX.test(filePath);
 
+/*
+ * Returns true if:
+ * filename is module.html, inside of *.module folder
+ */
+const isModuleHTMLFile = filePath => MODULE_HTML_EXTENSION_REGEX.test(filePath);
```

Comment:
Should these module utils live in the `templates.js` util file?

Reply:
that's a good point